enhancements/builds/disable-jenkins-pipeline.md

```diff
@@ -0,0 +1,249 @@
+---
+title: disable-jenkins-pipeline
+authors:
+  - "@adambkaplan"
+reviewers:
+  - @gabemontero
+  - Akram
+approvers:
+  - "@bparees"
+  - Jessica Forrester
+  - "@sbose78"
+creation-date: 2021-04-21
+last-updated: 2021-04-23
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Disable Jenkins Pipeline Strategy
+
+**Note - this enhancement is provisional and is intended to solicit feedback**
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenShift 3.0 included a way to invoke a Jenkins pipeline from an OpenShift build.
+The status of the build on the OpenShift cluster would reflect the state of the pipeline in Jenkins.
```

Comment:
technically speaking, the jenkins pipeline status is stored as encoded json in annotations in the build object

see https://github.com/openshift/jenkins-sync-plugin/blob/master/src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

the console then unpacks that json to provide the visualization it does for pipeline strategy

Reply:
@spadgett was this console view an OCP 3-ism? Did we get rid of the pipeline view for Jenkins in OCP 4?

Reply:
I think, yes, in probably because I don't recollect seeing it anywhere.

Reply:
@adambkaplan We added it in 4.x as well for parity. It looks like it's still there.

https://github.com/openshift/console/blob/master/frontend/public/components/build-pipeline.tsx